packages/0x.js/src/fetchers/simple_balance_and_proxy_allowance_fetcher.ts

```diff
@@ -0,0 +1,26 @@
+import { BlockParamLiteral } from '@0xproject/types';
+import { BigNumber } from '@0xproject/utils';
+
+import {BalanceAndProxyAllowanceFetcher} from '../abstract/balance_and_proxy_allowance_fetcher';
+import {TokenWrapper} from '../contract_wrappers/token_wrapper';
+
+export class SimpleBalanceAndProxyAllowanceFetcher implements BalanceAndProxyAllowanceFetcher {
+    private _token: TokenWrapper;
+    private _defaultBlock: BlockParamLiteral;
+    constructor(token: TokenWrapper, defaultBlock: BlockParamLiteral) {
+        this._token = token;
+        this._defaultBlock = defaultBlock;
+    }
+    public async getBalanceAsync(tokenAddress: string, userAddress: string): Promise<BigNumber> {
+        const methodOpts = {
+            defaultBlock: this._defaultBlock,
+        };
+        return this._token.getBalanceAsync(tokenAddress, userAddress, methodOpts);
```

Comment:
Our convention is to assign to a variable before returning. This helps with readability of exactly what is being returned.